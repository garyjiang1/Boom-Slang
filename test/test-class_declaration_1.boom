class MyObject:
	static:
		int x = 5
		string foo = "bar"


	required:
		int z
		float fOOOO



	optional:
		boolean boo = true

MyObject mo = MyObject(10, 1.0)
println(mo.z)
println(mo.fOOOO)
