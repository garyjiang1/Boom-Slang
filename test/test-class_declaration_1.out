10
1.0000
